Resources/Prototypes/Entities/Mobs/NPCs/silicon.yml

```diff
@@ -110,50 +112,34 @@
   - type: GuideHelp
     guides:
     - Robotics
-
+  - type: Speech
+    speechVerb: Robotic
+  - type: TypingIndicator
+    proto: robot
+    
 - type: entity
   parent: MobSiliconBase
-  id: MobTaxiBot
-  name: taxibot
-  description: Give a ride?
+  id: MobSiliconBaseVehicle # for vehicles
+  abstract: true
   components:
   - type: NoSlip
   - type: Sprite
-    drawdepth: Mobs
-    sprite: Mobs/Silicon/Bots/taxibot.rsi
-    layers:
-    - state: taxibot
-      map: ["enum.VehicleVisualLayers.AutoAnimate"]
     noRot: true
```

Comment:
This should be on the base silicon (realistically it should be on BaseMob and the silicon base should inherit it).